Web/HTML/html-structure/attributes.md

```diff
@@ -12,16 +12,34 @@ category: must-know
 tags:
   - introduction
 
+---
+# Attributes
 ---
 ## Content
 
-New content to go here. The author must be updated to match a valid Enki account.
+In HTML, elements can have attributes. They are placed in the opening tag and are there to provide extra information. 
+They usually come in name/value pairs and are enclosed in quotation marks.
+
+Example:
+```
+<img src="enki_logo.jpg">
+```
+In the example above the `src="enki_logo.jpg"` is an attribute for the `<img>` tag.
+
 
 ---
 ## Practice
 
 What are the two attributes of this code?
-```<p>This is a <a href="https://www.enkipro.com" target="_blank">link</a></p>```
+```
```

Comment:
This question is confusing with the two ??? slots, because if they do the correct answers in the wrong order, it's still wrong. Either remove one answer or note that it should be in order of appearance